balarco/urls.py

```diff
@@ -21,4 +21,5 @@
     url(r'^api/clients/', include('clients.urls')),
     url(r'^api/users/', include('users.urls', namespace='users')),
     url(r'^api/works/', include('works.urls', namespace='works')),
+    url(r'^api-auth/', include('rest_framework.urls', namespace='rest_framework')),
```

Comment:
I realized why I had it, it's for login functionality in the Django REST framework to manually test the Web Services